data/txHandler.go

```diff
@@ -263,19 +281,41 @@ func (handler *TxHandler) checkAlreadyCommitted(tx *txBacklogMsg) (processingDon
 		logging.Base().Debugf("txPool rejected transaction: %v", err)
 		return true
 	}
+	return false
+}
 
-	// build the transaction verification context
-	latest := handler.ledger.Latest()
-	latestHdr, err := handler.ledger.BlockHdr(latest)
-	if err != nil {
-		logging.Base().Warnf("Could not get header for previous block %v: %v", latest, err)
-		return true
-	}
-	tx.proto = config.Consensus[latestHdr.CurrentProtocol]
-	tx.spec.FeeSink = latestHdr.FeeSink
-	tx.spec.RewardsPool = latestHdr.RewardsPool
+// note: this modifies tx
+func (handler *TxHandler) prepareVerifyContexts(tx *txBacklogMsg, latestHdr bookkeeping.BlockHeader) error {
+	// TODO set verifyContexts correctly
```

Comment:
What does this comment mean?

Reply:
Ah, this comment is outdated--I forgot to remove it.